pkg/controller/controller_clusterservicebroker.go

```diff
@@ -38,9 +38,9 @@ import (
 // be easily combined with a follow on specific message.
 const (
 	errorFetchingCatalogReason  string = "ErrorFetchingCatalog"
-	errorFetchingCatalogMessage string = "Error fetching catalog. "
+	errorFetchingCatalogMessage string = "Error fetching catalog."
```

Comment:
Note the trailing spaces; those are intentional because these strings are use to construct error messages. Would you restore the trailing spaces?

Reply:
:+1: Didn't realize those were deliberate; that makes sense.